internal/cmd/gocdk/_static/biome_add/secrets.auto.tfvars

```diff
@@ -0,0 +1,6 @@
+# This would be where you would put secrets like database passwords.
```

Comment:
Nit: consider `biome` instead of `biome_add`; are any other biome commands going to need static templates? If so, maybe subdirectories (e.g., `biome/add`).

Reply:
Related to @zombiezen's [comment](https://github.com/google/go-cloud/pull/2131#discussion_r285202267) about having suggested `biome_add.go` since `biome.go` already had a bunch of stuff in it, and I wanted to name it to match the name of the command file.

Reply:
I meant the directory name under `static` here, e.g. `_static/biome/...` or `_static/biome/add/...` instead of `_static/biome_add/...`.